### PR TITLE
Fix #1686 -- Remove reference to Django core team

### LINKED ISSUE #1686: Remove reference to Django core team
While reviewing #1672 I noticed that there was a page still referencing the Django core team: https://github.com/django/djangoproject.com/blob/d786988b9a64185c71582ffc26c946ea084fe3bf/djangoproject/templates/conduct/changes.html#L18

The Django core team was dissolved a while back, so the text of this page and any others (`git grep -i "core team"` seems to yield other files too) should be updated.

### PR DESCRIPTION
**Pull request by ontowhee**
Fix for #1686 

Removed references to the "core team" on the pages "Code Of Conduct - Changes" and "Diversity - Changes".

Before:
Screenshot 2024-11-13 at 11 19 17 AM

After:
Screenshot 2024-11-13 at 11 19 40 AM

### CODE REVIEW

File: djangoproject/templates/conduct/changes.html [hunk 1]
```diff
@@ -15,11 +15,9 @@ <h2>Change control process</h2>
     proposed via a pull request to the
     <a href="https://github.com/django/djangoproject.com">djangoproject.com repository
       on GitHub</a>. Changes will be reviewed by the conduct committee first, and then
-    sent to the DSF, the Django core team, and the Django community for comment.
+    sent to the DSF and the Django community for comment.
     We'll hold a comment period of at least one week,  and then each group will vote
-    on the change using its normal process (a board for the DSF,
-    <a href="{% url 'document-detail' lang='en' version='dev' url='internals/contributing/bugs-and-features' host 'docs' %}#how-we-make-decisions">
-      a core dev vote</a> for the core team).
+    on the change using its normal process (a board for the DSF).
```
Comment (bmispelon) [inline djangoproject/templates/conduct/changes.html:20]: I think the sentence can be simplified now, maybe something like:

> We'll hold a comment period of at least one week, then the DSF board will vote on the change.

File: djangoproject/templates/diversity/changes.html [hunk 2]
```diff
@@ -13,11 +13,9 @@ <h2>Change control process</h2>
     to the
     <a href="https://github.com/django/djangoproject.com">djangoproject.com repository
       on GitHub</a>. Changes will be reviewed by the membership first, and then
-    sent to the DSF, the Django core team, and the Django community for comment.
+    sent to the DSF and the Django community for comment.
     We'll hold a comment period of at least one week,  and then each group will vote
-    on the change using its normal process (a board for the DSF,
-    <a href="{% url 'document-detail' lang='en' version='dev' url='internals/contributing/bugs-and-features' host 'docs' %}#how-we-make-decisions">
-      a core dev vote</a> for the core team).
+    on the change using its normal process (a board for the DSF).
```
Comment (bmispelon) [inline djangoproject/templates/diversity/changes.html:18]: Same comment as above

**Review by bmispelon — COMMENTED** [review]
Hi and thanks for looking into this.

I think the wording can be further simplified, I've left some suggestions. Let me know what you think 🎸

File: djangoproject/templates/conduct/changes.html (hunk 1 above)
Comment (ontowhee) [reply djangoproject/templates/conduct/changes.html]: Thanks for the suggestion! I agree, it is much simpler now. It was awkward with just one group mentioned in the parenthesis.

**Comment by bmispelon** [comment]
Excellent work, thank you 👍🏻
(The change should be deployed within the hour)

**Pull request merged**